apps/condo/domains/common/utils/codegeneration/generate.hooks.ts

```diff
@@ -0,0 +1,220 @@
+import { useCallback } from 'react'
```

Comment:
?? why you don't use existing utilites

Reply:
It is a redundant file slipped from somewhere here